lib/app/routes/exercises.rb

```diff
@@ -40,12 +40,14 @@ class Exercises < Core
           submission: submission,
           next_submission: next_submission,
           sharing: Sharing.new,
-          solution: src_obj.solution
+          solution: src_obj.solution,
         }
         erb :"submissions/show", locals: data
       end
 
       post '/submissions/:key/like' do |key|
+        require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.